libs/mini-rx-store/src/lib/utils.ts

```diff
@@ -1,69 +1,6 @@
-import { Observable, OperatorFunction, pipe } from 'rxjs';
-import { distinctUntilChanged, filter, map } from 'rxjs/operators';
-import {
-    Action,
-    EFFECT_METADATA_KEY,
-    HasEffectMetadata,
-    MetaReducer,
-    Reducer,
-    StateOrCallback,
-} from './models';
-import { isComponentStoreSetStateAction, isFeatureStoreSetStateAction } from './actions';
-import { miniRxNameSpace } from './constants';
+import { pipe } from 'rxjs';
+import { distinctUntilChanged, map } from 'rxjs/operators';
 
-export function ofType(...allowedTypes: string[]): OperatorFunction<Action, Action> {
-    return filter((action: Action) =>
-        allowedTypes.some((type) => {
-            return type === action.type;
-        })
-    );
-}
 export function select<T, R>(mapFn: (state: T) => R) {
```

Comment:
I think that the `select` function is only used in state.ts.

Following the collocation principle we could move the `select` function to state.ts and remove the utils.ts file completely.